contrib/upstart/cilium-net-daemon.conf

```diff
@@ -1,2 +1,2 @@
 env PATH=/usr/local/clang/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/sbin:/sbin:/bin
-exec cilium -D daemon run -d eth1
+exec cilium -D daemon run --ipv4 --lb
```

Comment:
So should k8s service integration assume the cilium is always running in LB mode? And what about `daemon/daemon.go:223`?